src/panels/config/config.js

```diff
@@ -19,6 +20,14 @@ export default {
     {
       id: 'options',
       buttons: [
+        {
+          active: true,
+          id: djs,
+          className: 'fa fa-file-code-o',
+          command: djs,
+          context: djs,
+          attributes: { title: 'Disable JS' }
+        },
```

Comment:
You can keep this button for the development purpose but in the end, I wouldn't include it